rslib/src/links.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright: Ankitects Pty Ltd and contributors
+// License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+
+use strum::{Display, EnumIter, EnumString};
+
+static HELP_SITE: &'static str = "https://docs.ankiweb.net/";
+
+#[derive(Debug, PartialEq, Clone, Copy, Display, EnumIter, EnumString)]
+pub enum HelpPage {
+    #[strum(serialize = "getting-started#note-types")]
+    Notetype,
+    #[strum(serialize = "browsing")]
+    Browsing,
+    #[strum(serialize = "browsing#find-and-replace")]
+    BrowsingFindAndReplace,
+    #[strum(serialize = "browsing#notes")]
+    BrowsingNotesMenu,
+    #[strum(serialize = "studying#keyboard-shortcuts")]
+    KeyboardShortcuts,
+    #[strum(serialize = "editing")]
+    Editing,
+    #[strum(serialize = "editing#adding-cards-and-notes")]
+    AddingCardAndNote,
+    #[strum(serialize = "editing#adding-a-note-type")]
+    AddingNotetype,
+    #[strum(serialize = "math#latex")]
+    Latex,
+    #[strum(serialize = "preferences")]
+    Preferences,
+    #[strum(serialize = "")]
+    Index,
+    #[strum(serialize = "templates/intro")]
+    Templates,
+    #[strum(serialize = "filtered-decks")]
+    FilteredDeck,
+    #[strum(serialize = "importing")]
+    Importing,
+    #[strum(serialize = "editing#customizing-fields")]
+    CustomizingFields,
+    #[strum(serialize = "deck-options")]
+    DeckOptions,
+    #[strum(serialize = "editing#features")]
+    EditingFeatures,
+}
+
+pub fn help_page_link(page: HelpPage) -> String {
+    format!("{}{}", HELP_SITE, page)
+}
+
+pub fn help_page_link_from_str(page: &str) -> String {
```

Comment:
We could try to parse the url here to report any errors. But since vanilla Anki should never use this, it seemed superfluous. Let me know if you think differently.